packages/lib/services/plugins/RepositoryApi.ts

```diff
@@ -165,24 +191,62 @@ export default class RepositoryApi {
 		}
 	}
 
-	public async search(query: string): Promise<PluginManifest[]> {
-		query = query.toLowerCase().trim();
+	public async searchAndFilter(category?: string, searchQuery?: string): Promise<PluginManifest[]> {
```

Comment:
rename back to just "search":

    public async search(query: string, options:SearchOptions = null): Promise<PluginManifest[]> {

then:

    interface SearchOptions {
         category?: ....
    }

Reply:
Category should be an enum